ros/src/msgs/autoware_msgs/msg/waypoint.msg

```diff
@@ -6,4 +6,23 @@ geometry_msgs/PoseStamped pose
 geometry_msgs/TwistStamped twist
 dtlane dtlane
 int32 change_flag
+
+uint32 lane_id
+uint32 left_lane_id
+uint32 right_lane_id
+uint32 stop_line_id
+
+# Lane Direction
+# FORWARD				= 0
+# FORWARD_LEFT	 		= 1
+# FORWARD_RIGHT			= 2
+# BACKWARD				= 3 
+# BACKWARD_LEFT			= 4
+# BACKWARD_RIGHT		= 5
+# STANDSTILL	 		= 6
+uint32 direction
+
+float32 cost
+float32 time_cost
+
```

Comment:
will this changes be back compatiable? At the very least it will break bags files that you already recorded...

Reply:
I need to check with other members of the team.